tcs/htf_props.cpp

```diff
@@ -91,6 +91,7 @@ bool HTFProperties::SetUserDefinedFluid( const util::matrix_t<double> &table )
 	// Set class member data
 	m_userTable = table;	
 	m_fluid = User_defined;
+    m_integration_points = 100;
```

Comment:
Does this default the user-defined HTF to 100 points? Does it make sense to potentially have more integration points than data points in the lookup table?

Reply:
No, it doesn't default the user-defined HTF table to 100 points. This is used in calculating the average Cp value across a arbitrary temperature range. I chose 100 arbitrarily. This could be more robust. This could be set by an optional input. Ideally, we would determined this dynamically depending on the number of points in the user table between the temperature range of the Cp calculation. However, this might take a bit of time to implement.